src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
+spider.style.position = 'static';
+spider.style.display = 'flex';
+spider.style.justifyContent = 'center';
+spider.style.alignItems = 'center';
+wall.style.display = 'flex';
+wall.style.alignItems = 'center';
+wall.style.justifyContent = 'center';
```

Comment:
please try to not modify the `position` and add the proper logic to absolute position